pulp_ansible/app/modelresource.py

```diff
@@ -132,6 +149,7 @@ class Meta:
     CollectionDeprecationResource,
     TagResource,
     CollectionVersionContentResource,
+    CollectionVersionMarkResource,
```

Comment:
Do you think, it would be possible to inject a mark or two into the import-export tests?